tools/meteor-services/deploy.js

```diff
@@ -96,11 +99,18 @@ function deployRpc(options) {
     Console.info("Talking to Galaxy servers at " + deployURLBase);
   }
 
+  let operand = '';
```

Comment:
Yeah I didn't feel great about calling everything a `site` – thanks for fixing this!